crates/typst-svg/src/lib.rs

```diff
@@ -1205,21 +1351,41 @@ impl SvgPathBuilder {
     }
 }
 
+impl Default for SvgPathBuilder {
+    fn default() -> Self {
+        Self(Default::default(), Ratio::one())
+    }
+}
+
 /// A builder for SVG path. This is used to build the path for a glyph.
 impl ttf_parser::OutlineBuilder for SvgPathBuilder {
     fn move_to(&mut self, x: f32, y: f32) {
+        let x = x * self.scale();
```

Comment:
I assume this is because of the patterns on text thing? But why does it make a difference to "inline" the transformation from glyph space rather than having it as a transform?

Code-wise, I think the multiplications could just be inline in the `write!` calls.

Reply:
Yes with the huge scales some readers were struggling to tile properly (i.e to fill the text with tiles) it's really unclear why but it's something I could find other reporting.